src/components/dialogs/copy-to-script-dialog.js

```diff
@@ -26,17 +27,25 @@ import PropTypes from 'prop-types';
  */
 const CopyToScriptDialog = ({
     id,
+    directoryUuid,
+    elementType,
+    onGenerateNameError,
     open,
     onClose,
     onClick,
     currentName,
     title,
 }) => {
-    const [newNameValue, setNewNameValue] = React.useState(currentName);
+    const [generatedName, setGeneratedName] = useState('');
+    const [newNameValue, setNewNameValue] = useState(currentName);
```

Comment:
Why we do use generatedName and newNameValue
Maybe one is enough

Reply:
because in that case (initialValue={newNameValue}) the useEffect with initialValue as a dependency will be triggered each time newNameValue is changed